### PR TITLE
feat: add doctypes

### PR DESCRIPTION
**Pull request by ptbrowne**
This brings the code from cozy-doctypes for working with doctypes
that are shared between several codebases here.

### CODE REVIEW

File: packages/doctypes/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "cozy-doctypes",
+  "version": "1.0.1-beta.6",
```
Comment (kosssi) [inline packages/doctypes/package.json]: On ne passerait pas en version stable ?

Comment (ptbrowne) [reply packages/doctypes/package.json]: done

File: packages/doctypes/src/Document.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
+const { omit, pick } = require('lodash')
```
Comment (kosssi) [inline packages/doctypes/src/Document.js]: On importe finalement tout `lodash` ici

Comment (ptbrowne) [reply packages/doctypes/src/Document.js]: done

**Pull request merged**